web-console/4.0-designs/resource-quota/resource-quota.md

```diff
@@ -0,0 +1,48 @@
+# Resource Quota
+
+![](img/resource-quota-details.png)
+
+## Compute Resources
+![](img/compute-resources.png)
+- Compute resource request and limit utilization are displayed at the top of the overview as gauges
+
+## Scopes
+![](img/Scopes.png)
+- Each scope applied to the resource quota should be visible in the overview along with a short description below. Descriptions should be as follows:
+- Best Effort
+  - `Affects pods that do not have resource limits set. These pods have a best effort quality of service.``
+- Not Best Effort
+  - `Affects pods that have at least one resource limit set. These pods do not have a best effort quality of service.`
+- Terminating
+  - `Affects pods that have an active deadline. These pods usually include builds, deployers, and jobs.`
+- Not Terminating
+  - `Affects pods that do not have an active deadline. These pods usually include your applications.`
+
+## Resource Quota Details
+![](img/quota-details.png)
+
+### Type
+The type column should the type of quota for each resource. `Request` for compute resource requests, `Limit` for compute resource limits, and `Count` for object counts.
```

Comment:
I'm not sure these three values cover all possibilities. Here is everything you can set:

https://docs.okd.io/latest/admin_guide/quota.html#managed-by-quota

Specifically there is quota on storage and image streams, which would need a different type. I'm not sure the tables in the doc have all possible values to be honest.

I'm worried this will be a source of bugs because we'll basically have to hard-code a "type" for every known quota resource, and new resources will be added over time.

Reply:
@spadgett Is there an alternative in your mind or are you thinking we need to ditch this column?

Reply:
I'd probably ditch the column. There isn't really a concept of type in quotas (and it might even be confused with scope).

If we keep it, we could use something like "Compute," "Storage," and "Objects." I worry this will be constantly getting out of date as new things are added. It looks to me like even our docs are incomplete. We're going to have the same problem in the code.

Reply:
@spadgett yeah, I don't feel super strongly about keeping that. I'm happy to get rid of it